Automattic/jetpack
# Extensions: Deduplicate help-message component

A nearly identical `HelpMessage` component existed in two places. Deduplicate. Noticed in https://github.com/Automattic/jetpack/pull/13072

#### Changes proposed in this Pull Request:

Deduplicate the nearly identical components.

#### Screens/HTML

##### Before

![Screen Shot 2019-07-17 at 13 27 29](https://user-images.githubusercontent.com/841763/61372454-72bc9380-a897-11e9-8d76-b5ec245b60da.png)

![Screen Shot 2019-07-17 at 13 29 18](https://user-images.githubusercontent.com/841763/61372493-8a941780-a897-11e9-8271-7b5671a7bb68.png)


```html
<div
	class="simple-payments__help-message simple-payments__help-message-is-error"
	id="0-title-error"
>
	<svg
		class="gridicon gridicons-notice-outline"
		height="24"
		width="24"
		xmlns="http://www.w3.org/2000/svg"
		viewBox="0 0 24 24"
	>
		<g>
			<path d="M12 4c4.41 0 8 3.59 8 8s-3.59 8-8 8-8-3.59-8-8 3.59-8 8-8m0-2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm1 13h-2v2h2v-2zm-2-2h2l.5-6h-3l.5 6z" />
		</g>
	</svg>
	<span>Please add a brief title so that people know what they’re paying for.</span>
</div>
```

##### After

![Screen Shot 2019-07-17 at 13 27 43](https://user-images.githubusercontent.com/841763/61372453-7223fd00-a897-11e9-810d-1291b9102924.png)

![Screen Shot 2019-07-17 at 13 30 08](https://user-images.githubusercontent.com/841763/61372496-8d8f0800-a897-11e9-87d6-41e14c23e5bd.png)

```html
<div class="help-message help-message-is-error" id="0-title-error">
	<svg
		class="gridicon gridicons-notice-outline"
		height="24"
		width="24"
		xmlns="http://www.w3.org/2000/svg"
		viewBox="0 0 24 24"
	>
		<g>
			<path d="M12 4c4.41 0 8 3.59 8 8s-3.59 8-8 8-8-3.59-8-8 3.59-8 8-8m0-2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm1 13h-2v2h2v-2zm-2-2h2l.5-6h-3l.5 6z" />
		</g>
	</svg>
	<span>Please add a brief title so that people know what they’re paying for.</span>
</div>
```

#### Testing instructions:

* Add a Simple Payment block
* Trigger validation error (leave title empty)
* Verify the help notice is visually unchanged.

#### Proposed changelog entry for your changes:

None.
